source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,356 @@
+~~~~~~~~~~~~~~~~
+
+You can revert a migration after it has completed and before it has been committed.
+
+Completed migrations are automatically committed in accordance with your 
+:atlas:`minimum oplog window </atlas/cluster-additional-settings/#set-minimum-oplog-window>`.
+You can increase or decrease the amount of time you have to 
+revert a migration by adjusting the minimum oplog window.
+
+You cannot revert a migration that has been committed.
+
+Similar to canceling a migration, if you revert a migration, you need to 
+start over from the beginning with future migration attempts.
+
+**To revert a migration:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, select the three dots next to 
+   :guilabel:`Manage Sync Migration`.
+#. From the list of options, select 
+   :guilabel:`Revert back to Partition-Based Sync`.
+   
+   .. image:: /images/revert-migration.png
+      :alt: Revert a migration
+   
+   This will revert your migration. You will need to migrate again if you 
+   want to enable Flexible Sync in the future.
+
+.. _realm-sync-migrate-after:
+
+After Backend Migration
+-----------------------
+
+Your migrated backend uses a partition key field in your Realm object models
+to map the Partition-Based Sync client objects to Flexible Sync equivalents on
+the backend. If your object models don't have a partition key field, the
+backend automatically injects one into every object the Partition-Based Sync
+client creates.
+
+These are some other characteristics of the injected field you should be aware
+of:
+
+- The injected field follows this structure: ``<yourPartitionKey> == 
+  <yourPartitionValue>``. The field key is derived from your old
+  Partition-Based Sync  **backend**  configuration. The value is derived from
+  your current Partition-Based Sync  **client**  configuration.
+- The injected field does not sync back to the client that created the object.
+  This means the originating client won't be able to read the injected
+  ``partitionKey`` field.
+
+The Partition-Based Sync client automatically creates sync subscriptions for 
+each table where ``partitionKey == partitionValue``. This will continue
+until you :ref:`migrate your client app <realm-sync-migrate-client>` to
+Flexible Sync.
+
+.. _realm-sync-migrate-client:
+
+Migrate Client App to Flexible Sync
+-----------------------------------
+
+After your app's backend has been migrated to Flexible Sync, we recommend
+that you also migrate your client app to Flexible Sync. When you migrate your
+client app to Flexible Sync, the client will stop automatically creating
+sync subscriptions for each Realm object model in your app's data model.
+
+We also recommend removing all sync subscriptions and then creating new subscriptions
+for your data. This is the clearest way to control what data is synced after a
+migration. Refer to the SDK documentation for details about adding and removing
+subscriptions.
```

Comment:
I don't know if you want to mention the naming format of the generated subscriptions while the client is in the migrated state. It follows this format: `flx_migrated_` followed by the name of the table/collection. For example, a subscription for the "Person" collection will be named: `flx_migrated_Person`